prow/resultstore/payload.go

```diff
@@ -242,12 +242,21 @@ func startedProperties(started *metadata.Started) []*resultstore.Property {
 	for _, r := range repos {
 		ps = append(ps, &resultstore.Property{
 			Key:   "Repo",
-			Value: "https://" + r,
+			Value: fixRepo(r),
 		})
 	}
 	return ps
 }
 
+// fixRepo deletes the "-review" segment present in some Gerrit Repos to
+// produce a valid URL. Cf. CreateRefs() in gerrit/adapter/adapter.go.
```

Comment:
Would it be a good idea to unify these two areas in a follow-up change?

Reply:
Seems we'd have to create a new package under gerrit to factor out one function to share. Unless there's some other place that makes sense for this kind of code. Open to suggestions.

Reply:
Note gerrit/source/source.go has CodeRootURL() which is similar but implements an assertion along with it that we would not want here.

Reply:
Ack. It might make sense to move it to a package that both areas already import (assuming such a package exists, I haven't checked).

I think it's fine to leave this as is if it's going to take a lot of effort to refactor.

Reply:
Our comments posted at the same time.

> Note gerrit/source/source.go has CodeRootURL() which is similar but implements an assertion along with it that we would not want here.

SG, thanks for looking into it.

Reply:
See https://github.com/kubernetes/test-infra/pull/31490